utils/cloud/gcp/gcloud_configure.yml

```diff
@@ -1,5 +1,5 @@
 - name: authenticate gcloud service account
-  shell: gcloud auth activate-service-account --key-file=/mnt/cloud_config.yml
+  shell: gcloud auth activate-service-account --key-file=/mnt/cloud_config/cloud_config.yml
```

Comment:
Are we not able to mount to path directly, i.e., /mnt/cloud_config.yml ? 

We are already creating this file in image build itself I suppose